<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PDFCreatorForm</class>
 <widget class="QWidget" name="PDFCreatorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>151</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0" colspan="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QCheckBox" name="textAntialiasingCheckBox">
     <property name="text">
      <string>Text antialising</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="textHintingCheckBox">
     <property name="text">
      <string>Text hinting</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="antialiasingCheckBox">
     <property name="text">
      <string>Antialiasing</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <connections/>
</ui>
